specifications/api/swagger.yaml

```diff
@@ -1879,6 +1895,28 @@ components:
         status: *transactionStatus
         metadata: *transactionMetadata
 
+    x-txBody: &txBody
+      oneOf:
+        - <<: *serializedHex
+          title: Serialized transaction body
+
+    x-txWits: &txWits
+      type: array
+      items: *serializedHex
+      title: Serialized transaction witnesses
+
+    ApiSerialisedTransaction: &ApiSerialisedTransaction
+      oneOf:
+        - <<: *serializedHex
+          title: Serialized transaction
+        - type: object
+          required:
+            - body
+            - wits
+          properties:
```

Comment:
How should I understand this? Either we get serialized hex which means all needed witnesses were constructed and added and the signed txs can be submitted OR I have partially signed tx, ie., not enough witnesses collected yet to treat it as fully signed tx?

Reply:
I added some more swagger doc strings.
The idea is that the user can submit an encoded tx, or an encoded tx body with zero or more witnesses.
The signing endpoint will add new witnesses to the set.